content/learn/05.communication/08.bluetooth/bluetooth.md

```diff
@@ -0,0 +1,530 @@
+
+**Connection Types**
+
+-	Bluetooth Classic: establishes a continuous and relatively power-hungry connection, making it suitable for applications requiring real-time, continuous communication.
+
+-	Bluetooth Low Energy: supports two primary modes - advertising and connection. In advertising mode, a BLE peripheral periodically broadcasts its presence but doesn't maintain a continuous connection, conserving power. When needed, a central device can establish a connection for data exchange.
+
+
+## ArduinoBLE Library
+
+The [ArduinoBLE Library](https://www.arduino.cc/reference/en/libraries/arduinoble/) is the main library enabling Bluetooth Low Energy on compatible Arduino boards. You must first download and install the ArduinoBLE library. See our [instructions](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library) on how to install a library. 
+
+In the following section you will find an overview and explanation of the librarie's most important methods and how to use them:
+
+- `BLE.begin()` - Initialize the library
+
+### Central Device
+
+- `BLE.scan()` - Scan for peripherals
+
+- `BLE.available()` - Check for available peripherals
+
+- `BLE.scanForUuid("UUID")` - Start scanning for peripherals with specific UUID
+
+- `bleDevice.address()` - Check the peripheral's address
+
+- `bleDevice.localName()` - Check the peripheral's name
+
+- `bleDevice.advertisedServiceUuid()` - Check the peripheral's advertised service
+
+- `bleDevice.connect()` - Connect to a Bluetooth Low Energy device.
+
+- `BLE.connected()` - Check if connection was successful
+
+- `bleDevice.discoverAttributes()` - Discover a peripheral's attributes
+
+- `BLE.stopScan()` - Stop scanning
+
+- `BLE.disconnect()` - Disconnect from a peripheral
+
+### Peripheral
+
+- `BLE.setLocalName(name)` - Give your device a name
+
+- `BLEService newService(service)` - Create a service object
+
+> Read more about standard services in the [Assigned Numbers document](https://www.bluetooth.com/specifications/assigned-numbers/).
+
+- `BLEUnsignedCharCharacteristic customCharacteristic("UUID_here", Properties_here);` - Create characteristic object
+
+- `BLE.setAdvertisedService(bleService)` - Set the advertised service
+
+- `bleService.addCharacteristic(bleCharacteristic)` - Add characteristics to the service
+
+- `bleCharacteristic.writeValue()` - Write the value of the characteristic
+
+- `BLE.advertise()` - Start advertising
+
+***These are just a few of the most important methods. You can find more information about all methods and their details in the [ArduinoBLE reference](https://www.arduino.cc/reference/en/libraries/arduinoble/).***
+
+## Examples
+
+Below you can find examples showing how to send data between two Arduino boards and how to connect to your Arduino board, reading and writing values using your smartphone.
+
+### Send data between two Arduino boards
+
```

Comment:
```suggestion

This example can be used with two Bluetooth® supported Arduino boards, one with a button, and the other with an LED. When the button is pressed, it will advertise a corresponding value (high/low), which will be received by the board with an LED.

```

Reply:
@Hannes7eicher just after this section, can we add a circuit diagram of the two board's setup? Basically the ones from the other tutorials, but compressed into one image if possible, like this:

```
#### Circuit

The following circuit is required for this example. Connect a button to one board (pin X), and an LED to the other (pin Y).

![Remote LED control circuit.](assets/image.png)

```

Reply:
Good idea!